### DISCUSSION TITLE
how to java object to json

### DISCUSSION THREAD

**Discussion by thlco** [ldr0629/Together#10] [Q&A] [answered]
how to java object to json?

**Comment by ldr0629** [comment, marked as answer]
To convert your object in JSON with Jackson:

import com.fasterxml.jackson.databind.ObjectMapper; 
import com.fasterxml.jackson.databind.ObjectWriter; 

ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
String json = ow.writeValueAsString(object);
